### ISSUE TITLE
Icon color is lost when changing icon

### ISSUE THREAD

**Issue by JamesVertisan** [FlorianWoelki/obsidian-iconize#366]
When changing the icon for a note or folder, the icon color is lost. This ends up being very frustrating because I have all of my content color-coded by topic.  It would  be helpful if the icon color was persisted regardless of which icon has been selected. There are also occassions where I find a better icon to represent a note, but alas, the color is lost when I switch the icon. 

Kindly resolve this.
Thanks

**Comment by FlorianWoelki** [comment]
i don't think that i am going to fix this. at the moment, we have to live with that because when we set a color of an icon, it gets saved inside the data. that means when you set a new icon, the colour gets automatically applied, which can confuse the user.
